modules/misunderstood/src/views/full/index.tsx

```diff
@@ -27,8 +45,36 @@ interface State {
   selectedEventIndex: number | null
   selectedEvent: FlaggedEvent | null
   eventNotFound: boolean
+  dateRange?: DateRange
 }
 
+const shortcuts: IDateRangeShortcut[] = [
+  {
+    dateRange: [thisWeek, now],
+    label: lang.tr('module.analytics.timespan.thisWeek')
```

Comment:
Well thought but this won't work if analytics module is disabled. either duplicate translations or move translations to ui-shared. any other cleaner way to do this @allardy  ?

Reply:
I suggest we move these to ui-shared. We could also move the dates definition to something like `utils.dates.dateRanges` so both modules use the same list of suggestions